Cargo.toml

```diff
@@ -25,6 +25,7 @@ tokio-runtime = ["tokio/macros", "tokio/net", "tokio/rt", "tokio/time", "serde_b
 async-std-runtime = ["async-std", "async-std/attributes", "async-std-resolver", "tokio-util/compat"]
 sync = ["async-std-runtime"]
 tokio-sync = ["tokio-runtime"]
+openssl-tls = ["openssl", "openssl-probe", "tokio-openssl"]
```

Comment:
can we update the feature flag table in the readme and `lib.rs` documentation to include this flag?

Reply:
Done.